mystyle.css

```diff
@@ -0,0 +1,79 @@
+body {
+  background-color: #fdfdfd;
+  margin: 0;
+  padding: 0;
+}
+
+nav {
+  width: 50%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+}
+
+.navbar img {
+  width: 20px;
+  margin-top: 20px;
+}
+
+section {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+h1 {
+  font-family: 'Playfair Display', serif;
+  font-size: 50px;
+  font-weight: 400;
+  line-height: 57px;
+  margin-left: 280px;
+  margin-top: 120px;
+}
+
+.paragraph {
+  display: flex;
+  width: 16%;
+  margin-left: 280px;
+  margin-top: -30px;
+}
+
+p {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 400;
+  font-size: 14px;
+}
+
+.rectangle {
+  margin-left: 16%;
+  margin-top: 35px;
+}
+
+.rectangle button {
+  background-color: #1a2236;
+  width: 310px;
+  height: 60px;
+  padding: 10px;
+  color: white;
+  font-size: 24px;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+}
+
+button:hover {
+  color: #ff7a59;
```

Comment:
- [ ] [REQUIRED] I suggest that you kindly change the color of the button on hover to #1A2236;

- [ ] [REQUIRED] I also kindly suggest that you add styles for the button when it's pressed